vcluster/configure/vcluster-yaml/sync/from-host/configmaps.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: ConfigMaps
+sidebar_label: configmaps
+sidebar_position: 10
+description: Configuration for ...
+---
+
+import EnableSwitch from '../../../../_partials/config/sync/fromHost/configMaps.mdx'
+import ProAdmonition from '../../../../_partials/admonitions/pro-admonition.mdx'
+import FromHostConfigMapExample from '../../../../_fragments/sync-from-host-configmap-example.mdx'
+
+By default, this is turned off.
+
+:::info No need to configure RBAC
+vCluster automatically adds the required cluster RBAC permissions for retrieving the config maps and syncing the resources from the host to the virtual cluster.
+:::
+
+Enabling this allows you to sync ConfigMaps from the specified namespaces in the host to the specified namespaces in the vCluster.
+
```

Comment:
Can you add a very minimal example here. That's a goal for all our docs pages, and then extended examples can live at the bottom.

Reply:
added